clients/privacy-center/__tests__/common/location.test.ts

```diff
@@ -0,0 +1,108 @@
+import { createRequest } from "node-mocks-http";
+
+import { getLocation } from "~/common/location";
+
+describe("getLocation", () => {
+  describe("when using location headers", () => {
+    it("returns location data from country & region headers", () => {
+      const req = createRequest({
+        url: "https://privacy.example.com/fides.js",
+        headers: {
+          "CloudFront-Viewer-Country": "US",
```

Comment:
just so I understand, would this only happen if the privacy center were running through cloudfront?

Reply:
Exactly, yup! I think you'd also need to configure a cache policy that forwards those headers, etc.

But what that means is that, by routing through the CloudFront CDN, we'd instantly have an accurate picture of the user's location before the bundle is even built. That allows us to server location-specific bundles via this API 👍 

We use CloudFront for Fides Cloud so we'd be able to try this out ourselves